### ISSUE TITLE
Investigate 1:00 → one o’clock story promo bug

### ISSUE THREAD

**Issue by dr3** [bbc/simorgh#2309]
**Describe the bug**
If an audio story has a duration of 1:00 it is announced as `one o'clock` instead of `one minute` or `1 colon 00`

**To Reproduce**

1. Use Talkback, Jaws or Read&Write
2. Hear error

**Expected behavior**
It doesn't say `one o'clock`, but instead an acceptable alternative like `one minute` or `1 colon 00`

**Devices/Tools:**
TalkBack (Screen Reader) 
Latest Version with Chrome (Latest Version) on Android (Latest Version)

JAWS (Screen Reader) 
Version 17 with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

Read&Write (Reading Solution) 
Latest Version with Internet Explorer 11 on Windows (XP/Vista/7/8/10)

**Additional context**

**Comment by greenc05** [comment]
We should apply the solution that is in the screen reader UX which is:

Given I use a screen reader
When the media duration is announced
Then this must be announced with units of measurement for time e.g. hours/minutes/seconds such as ‘1 minute 33 seconds’

https://github.com/bbc/simorgh/issues/2085

**Comment by j-pendlebury** [comment]
Investigate if there is an easy solution for this. 

If not try using full English text, i.e. 1 minute 23 seconds

**Comment by greenc05** [comment]
Note text should be in the local language @jblaut "If not try using full **English** text, i.e. 1 minute 23 seconds" e.g. minutes and seconds should be the language of the page.

**Comment by DenisHdz** [comment]
This issue can be closed, as the `MediaIndicator` should not be announced by screen readers, and we will be replacing the colons with commas in the promo headline hidden duration, for the time being https://github.com/bbc/simorgh/issues/2996.
